### ISSUE TITLE
Option to keep read notifications

### ISSUE THREAD

**Issue by puresick** [rafaelmardojai/forge-sparks#29]
Sometimes I read notifications but want to keep them to visit them at a later point in time.
It would be helpful to have the read notifications stay visible with a manual "dismiss" action button. This could be something configurable in the settings to keep the current behaviour intact aswell.

Would something like this be in scope for Forge Sparks or is this something against this projects goal? :)

**Comment by rafaelmardojai** [comment]
I think GitHub (Gitea too I think) would mark the notification as read when you open the link.

That would require some mechanism in the app to mark the notification as unread.

**Comment by puresick** [comment]
Yes. GitHub has also a dedicated "Mark as done" functionality which removes the notification item from their notification area.

Looking at Forgejo they only have read and unread states.

**Comment by rafaelmardojai** [comment]
> Yes. GitHub has also a dedicated "Mark as done" functionality which removes the notification item from their notification area.

Yes, but their API doesn't have any way to list "not done" notifications, only read and unread.
